jobHandler/injected/run.py

```
@@ -0,0 +1,200 @@
+                await send_to_worker(workers[message['type']], message)
+
+    async def handle_request(websocket, message):
+        logger.info(23)
+        aspAdliVarLog(websocket, 18)
+        aspAdliVarLog(message, 19)
+        logger.info(24)
+        '\n        Handle incoming requests and pass to relevant worker.\n    '
+        logger.info(25)
+        if message['type'] in workers:
+            logger.info(26)
+            await workers[asp_temp_var_cc09d373046111f0b89d9f0ee9e28e0e].send(json.dumps(message))
```

Ruff (0.8.2)

Undefined name `asp_temp_var_cc09d373046111f0b89d9f0ee9e28e0e`

(F821)